### DISCUSSION TITLE
文本被检测框拼接了，这种需要自己训练模型么？

### DISCUSSION THREAD

**Discussion by hujiachun** [PaddlePaddle/PaddleOCR#13247] [Q&A]
#### 问题描述 / Problem Description
![test014](https://github.com/PaddlePaddle/PaddleOCR/assets/17762932/801bba08-c6e0-4b0e-88c9-075fbb2485e0)

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/17762932/86ca2bf5-b3ac-4f4e-a252-553d77d89cb2)

#### 运行环境 / Runtime Environment
- OS:Linux
- Paddle:2.6.1
- PaddleOCR:2.7.3

**Comment by GreatV** [comment]
换其他模型试试

**Comment by hujiachun** [comment]
> 换其他模型试试

v3 v4的各种版本都试过

**Comment by GreatV** [comment]
@hujiachun  那你得自己训练一个检测模型了
